src/components/data_comps/datm/cime_config/config_component.xml

```diff
@@ -59,22 +59,15 @@
   <entry id="DATM_PRESAERO">
     <type>char</type>
     <valid_values>none,clim_1850,clim_2000,trans_1850-2000,rcp2.6,rcp4.5,rcp6.0,rcp8.5,cplhist</valid_values>
-    <default_value>none</default_value>
+    <default_value>clim_2000</default_value>
     <values match="last">
-      <value compset="^1850_">clim_1850</value>
-      <value compset="^2000_">clim_2000</value>
-      <value compset="^2003_">clim_2000</value>
-      <value compset="^4804_">clim_2000</value>
       <value compset="^1850_">clim_1850</value>
       <value compset="^RCP8_">rcp8.5</value>
       <value compset="^RCP6_">rcp6.0</value>
       <value compset="^RCP4_">rcp4.5</value>
       <value compset="^RCP2_">rcp2.6</value>
       <value compset="^HIST_">trans_1850-2000</value>
       <value compset="^20TR_">trans_1850-2000</value>
-      <value compset="_DICE.*_POP2">none</value>
-      <value compset="_DLND.*_DICE.*_DOCN">none</value>
-      <value compset="_SLND.*_DICE.*_DOCN">none</value>
       <value compset="_DATM%CPLHIST">cplhist</value>
```

Comment:
Is this set of changes part of points 1-5 in the PR Comment?  If not, should it be mentioned separately?

Reply:
Good catch. I have added an additional comment section 6 in the PR to clarify this.

Reply:
Thanks!